Assets/Scripts/UI/Pickup Pup/PPAdoptionUIController.cs

```diff
@@ -4,6 +4,7 @@
  */
 
 using UnityEngine;
+using UnityEngine.UI;
```

Comment:
This "using" statement is not needed in the class, based on the libraries it currently uses.